src/TimeWarpEventDispatcher.cpp

```diff
@@ -52,7 +54,8 @@ TimeWarpEventDispatcher::TimeWarpEventDispatcher(unsigned int max_sim_time,
     std::unique_ptr<TimeWarpTerminationManager> termination_manager,
     std::unique_ptr<TimeWarpStatistics> tw_stats) :
         EventDispatcher(max_sim_time), num_worker_threads_(num_worker_threads),
-        is_lp_migration_on_(is_lp_migration_on), 
+        is_lp_migration_on_(is_lp_migration_on),
```

Comment:
why is the is_lp_migration_on_ line 57 classified as changed ? Please make sure there are no tabs in code or whitespaces after the comma

Reply:
No white spaces, it probably marked it as changed since I was struggling to get k0 and k1 to construct properly. I made a lot of different edits at the time.